app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for i in range(len(phrase)):
+        if phrase[i].lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
as more compact solution
```suggestion
    phrase.lower().count(letter.lower())
```